contracts/interfaces/GnosisIERC20.sol

```diff
@@ -0,0 +1,11 @@
+pragma solidity ^0.5.0;
+
+import "./GnosisIERC20Info.sol";
+
+import "openzeppelin-solidity/contracts/token/ERC20/IERC20.sol";
+
+/// @title Abstract Contract GnosisIERC20
+/// @dev Inherits from OpenZeppelin IERC20
+/// @dev Interface for full ERC20 functions + constants: token name, symbol, decimal
+// TODO: Convert to proper interface when/if inheritance changes incorporated
+contract GnosisIERC20 is GnosisIERC20Info, IERC20 {}
```

Comment:
OpenZeppelin also has a similar contract called [ERC20Detailed](
https://github.com/OpenZeppelin/openzeppelin-solidity/blob/master/contracts/token/ERC20/ERC20Detailed.sol). It's got an implementation though, but if you cast an address to it, Solidity will treat like an interface. Using it as a TruffleContract would be trickier, since it actually has an interface and would thus emit bytecode, and truffle-contract would check the bytecode before allowing you to instantiate it if bytecode was present in an artifact. Just FYC.

Reply:
interesting, thanks for this